### ISSUE TITLE
Remove perl from samples 

### ISSUE THREAD

**Issue by rgregg** [knative/docs#245]
## Expected Behavior
Samples are clear and easy to follow and walk you through the code to deploy the sample.

## Actual Behavior
We use perl in a few samples to do a string replacement in the yaml file. While beneficial, it doesn't help newcomers understand what's happening and we should replace that step with the actual changes to make to the yaml.

## Steps to Reproduce the Problem

- [ ] telemetry-go/README.md
- [ ] rest-api-go/README.md
- [ ] knative-routing-go/README.md
- [ ] autoscale-go/README.md
- [ ] thumbnailer-go/README.md
- [ ] grpc-ping-go/README.md
- [ ] buildpack-function-nodejs/README.md
- [ ] buildpack-app-dotnet/README.md
- [ ] gitwebhook-go/README.md

**Comment by tylerauerbeck** [comment]
@rgregg Not sure if this has just been addressed since you opened the issue -- but it appears that in the few files that I've checked this is already taken care of. The perl one liner is just offered as an option to do this quickly.

Example:
```
Manually replace:
image: github.com/knative/docs/serving/samples/rest-api-go with image: <YOUR_CONTAINER_REGISTRY>/serving/samples/rest-api-go
```
Is the above example sufficient enough to explain what actually needs to be done? Or are you looking for something more substantial? If this is fine, do we just want to remove the perl command?
